src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableClient.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.ContextInterfaces
+{
+    /// <summary>
+    /// Provides functionality available to durable orchestration and entity clients.
+    /// </summary>
+    public interface IDurableClient : IDurableOrchestrationClient, IDurableEntityClient
+    {
+        /// <summary>
+        /// Gets the name of the task hub configured on this client instance.
+        /// </summary>
+        /// <value>
+        /// The name of the task hub.
+        /// </value>
+        string TaskHubName { get; }
```

Comment:
This gives me a warning because it hides the other interfaces' TaskHubName. I added this in IDurableClient so there wasn't ambiguity when referencing TaskHubName. What is the best way to deal with this?

Reply:
It looks like using the `new` keyword [can help you get around this.](https://stackoverflow.com/a/16322323/9035640)